test/spe_test.py

```diff
@@ -39,8 +41,8 @@ def test_dir_and_file_created(self):
         if os.path.exists(self.persisted_dir):
             shutil.rmtree(self.persisted_dir)
         self.assertEqual(False, os.path.exists(self.persisted_dir))
-        date_holder = AssignmentLatestSubmittedDate(self.persisted_dir, self.file_name)
-        latest_test_taken_date = date_holder.get_assign_submitted_date()
+        # date_holder = AssignmentLatestSubmittedDate(self.persisted_dir, self.file_name)
```

Comment:
since data_holder is already defined as self variable, this commented line and others can be removed

Reply:
I missed removing then, Done!